src/relay/op/memory/memory.h

```diff
@@ -35,6 +35,7 @@ namespace tvm {
 namespace relay {
 
 Expr AllocStorage(Expr size, Expr alignment, SEScope se_scope, DataType dtype_hint);
+const Op& MemoryAllocTensorOp();
```

Comment:
This looks kind of odd, can you provide a comment at least?

Reply:
Oh it snuck in from the main change, sorry. Added comment, just trying to cut down on the duplicated special operator name strings.